app/partials/desktop/im.html

```diff
@@ -206,7 +206,7 @@ <h3 class="im_dialogs_empty_header" my-i18n="im_get_started"></h3>
 
                 <div class="im_send_form_wrap clearfix" ng-controller="AppImSendController" ng-class="{im_send_reply_form_wrap: draftMessage.replyToMsgID > 0}">
 
-                  <a class="pull-right im_panel_peer_photo" my-peer-photolink="historyPeer.id" img-class="im_panel_peer_photo" watch="true" ng-show="!draftMessage.isBroadcast">
+                  <a class="pull-right im_panel_peer_photo" my-peer-photolink="historyPeer.id" img-class="im_panel_peer_photo" watch="true" ng-show="!draftMessage.isBroadcast" ng-if="!historyPeer.self">
```

Comment:
Instead of just hiding it completely would be better to show "Saved messages" (and open profile on click?)